src/applications/personalization/connected-accounts/components/AppDeletedAlert.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+
+export class AppDeletedAlert extends React.Component {
+  componentDidMount() {
+    this.alert.focus();
+  }
+
+  alertMessage() {
+    // eslint-disable-next-line prettier/prettier
+    return `${this.props.account.attributes.title} has been disconnected from your account.`;
+  }
+
+  render() {
+    return (
+      <div
+        tabIndex="-1"
+        ref={alert => {
```

Comment:
This callback `ref` is working exactly as we want. I've had one case in the past where I had to write it as a bound class method to prevent the `ref` being called twice: https://reactjs.org/docs/refs-and-the-dom.html#caveats-with-callback-refs

Let's not refactor unless we see unexpected behavior in JAWS and/or NVDA. I will test in Staging when ready.